### DISCUSSION TITLE
Ubuntu 24.04 CI runs taking significantly longer then Ubuntu 22.04 CI runs

### DISCUSSION THREAD

**Discussion by gavintoddchurch** [community/community#160582] [Actions] [answered]
We update our images monthly based off of main of this repo and last month we updated to 24.04 and our pipelines were taking anywhere from 2x-4x longer then normal. I figured it was just a bad image but we attempted again this month and the same thing.

We've seen this across multiple runtimes (java, javascript, .net, python, etc.)

Is anyone else experiencing this?

**Comment by vidyasagarnimmagaddi** [comment, marked as answer]
Hi @gavintoddchurch 
We the runner images , Provides all always the best services to our customers, Apologies for the problem you have faced. 
Please feel free to specify the problem your facing by raising an issue in our public repo, so that an proper solution is provided, thanks

**Reply by Kiddahkisz** [reply]
#

**Comment by Ami-007** [comment]
> We update our images monthly based off of main of this repo and last month we updated to 24.04 and our pipelines were taking anywhere from 2x-4x longer then normal. I figured it was just a bad image but we attempted again this month and the same thing.
> 
> We've seen this across multiple runtimes (java, javascript, .net, python, etc.)
> 
> Is anyone else experiencing this?

Our team is also experiencing this issue. We moved over to using this OS in the last few days. Are you still experiencing this issue @gavintoddchurch ?

**Reply by gavintoddchurch** [reply]
We attempted to move to this image for 3 months in a row and saw the same thing every time. I decided to stop attempting to move to the new OS until last half of the year.

@vidyasagarnimmagaddi I didn't think creating an issue was appropriate since i don't have a specific methodology to do this. I'll try to create new images in April with the new OS and see if i'm seeing the same results.

I would need to test again but at one point I was able to reproduce this occasionally by just using the hosted runners in Azure DevOps

**Reply by Kiddahkisz** [reply]
#10007

**Reply by Ami-007** [reply]
Thanks for the reply. I appreciate the info and it’s good to know others have also or are also facing this issue.

**Reply by Ami-007** [reply]
As an fyi to the op and anyone else who may discover this, I have raised an issue. Please see here: https://github.com/actions/runner-images/issues/11790#issuecomment-2733868737
